packages/firebase_analytics/lib/firebase_analytics.dart

```diff
@@ -92,7 +92,7 @@ class FirebaseAnalytics {
   /// https://firebase.google.com/docs/reference/android/com/google/firebase/analytics/FirebaseAnalytics.html#setCurrentScreen(android.app.Activity, java.lang.String, java.lang.String)
   /// https://firebase.google.com/docs/reference/ios/firebaseanalytics/api/reference/Classes/FIRAnalytics#setscreennamescreenclass
   Future<Null> setCurrentScreen(
-      {@required String screenName, String screenClassOverride}) async {
+      {@required String screenName, String screenClassOverride: 'Flutter'}) async {
```

Comment:
Small nit that I also saw in another PR today: is it Flutter style to put spaces on the inside of curly braces? I realize that this file is formatted without these spaces so feel free to leave it as is for consistency if you don't want to reformat the file.